smsdk/client.py

```diff
@@ -445,13 +505,27 @@ def get_cookbook_top_results(self, recipe_group_id, limit=10, **kwargs):
             recipe_group_id, limit, **kwargs
         )
 
-    def get_cookbook_current_value(self, variables=[], minutes=1440, **kwargs):
+    def get_cookbook_current_value(self, *args, **kwargs):
```

Comment:
=None the args

Reply:
This is the standard way of defining positional arguments. If there are no positional arguments passed, it will be come None. No need to set it explicitly.